src/components/common.js

```diff
@@ -39,24 +40,15 @@ export const Clickable = ({ as = 'div', disabled, tooltip, tooltipSide, onClick,
   }
 }
 
-const linkProps = ({ disabled, variant }) => ({
-  as: 'a',
-  style: {
-    color: disabled ? colors.dark(0.7) : colors.accent(variant === 'light' ? 0.3 : 1),
-    cursor: disabled ? 'not-allowed' : 'pointer',
-    fontWeight: 500
-  },
-  hover: disabled ? undefined : { color: colors.accent(variant === 'light' ? 0.1 : 0.8) }
-})
-
-export const Link = ({ onClick, href, disabled, variant, children, ...props }) => {
+export const Link = ({ disabled, variant, children, ...props }) => {
   return h(Clickable, _.merge({
-    ...linkProps({ disabled, variant }),
-    href, disabled,
-    onClick: href && onClick ? e => {
```

Comment:
There are several places where we want to both navigate to another page, and also e.g. close a menu. There seem to be many more cases of that, so that seemed like a strong signal for keeping the default as is.